src/shadow-embed.js

```diff
@@ -78,13 +81,24 @@ export function createShadowRoot(hostElement) {
         },
       });
     }
-    return shadowRoot;
   } else if (shadowDomSupported == ShadowDomVersion.V0) {
-    return hostElement.createShadowRoot();
+    shadowRoot = hostElement.createShadowRoot();
+  } else {
+    shadowRoot = createShadowRootPolyfill(hostElement);
+  }
+
+  if (!isShadowCssSupported()) {
+    const rootId = `i-amphtml-sd-${win.Math.floor(win.Math.random() * 10000)}`;
```

Comment:
Which part of this is already done inside the `createShadowRootPolyfill`? Should it be moved here?

Reply:
See my answer to the shadow-embed comment. There was a duplicate call, and kept it to here, since we are calling `createShadowRootPolyfill` due to the shadowDom Version, and not because of the fact if shadow css is supported or not